lib/plugins/a11y.js

```diff
@@ -0,0 +1,30 @@
+export default shower => {
+    const liveRegion = document.createElement('section');
+    liveRegion.className = 'region';
+    liveRegion.setAttribute('role', 'region');
+    liveRegion.setAttribute('aria-live', 'assertive');
+    liveRegion.setAttribute('aria-relevant', 'all');
```

Comment:
Previous value: `additions`. Settings `innerHTML` triggers both additions & removals, so let's be explicit about this.